src/lib/hashes-combined.ts

```diff
@@ -0,0 +1,41 @@
+import { Poseidon } from './hash.js';
+import { Keccak } from './keccak.js';
+import { Bytes } from './provable-types/provable-types.js';
+
+export { Hash };
+
+// TODO do we want this?
```

Comment:
I am not sure anymore, tbh :D we proposed this in the RFC many many months ago, which is now severely outdated. I think it doesn't hurt, at least for now